src/components/ebay-progress-spinner/README.md

```diff
@@ -0,0 +1,11 @@
+# ebay-progress-spinner
+
+```marko
+<ebay-progress-spinner size="large" />
+```
+
+## Attributes
```

Comment:
@austingardner Sorry, I forgot to mention that we'll also need an `a11y-text` attribute that is used as the value for the aria-label.